Dockerfile

```diff
@@ -33,6 +33,11 @@ RUN curl -o /usr/bin/rancherd -sfL "https://github.com/rancher/rancherd/releases
 
 # Download nerdctl
 ARG NERDCTL_VERSION=1.2.1
-RUN curl -o ./nerdctl-bin.tar.gz -sfL "https://github.com/containerd/nerdctl/releases/download/v${NERDCTL_VERSION}/nerdctl-${NERDCTL_VERSION}-linux-amd64.tar.gz"
+RUN ARCH=$(uname -a); \
+    if [ "${ARCH}" == "x86_64" ]; then \
+    curl -o ./nerdctl-bin.tar.gz -sfL "https://github.com/containerd/nerdctl/releases/download/v${NERDCTL_VERSION}/nerdctl-${NERDCTL_VERSION}-linux-amd64.tar.gz"; \
+    else \
+    curl -o ./nerdctl-bin.tar.gz -sfL "https://github.com/containerd/nerdctl/releases/download/v${NERDCTL_VERSION}/nerdctl-${NERDCTL_VERSION}-linux-arm64.tar.gz"; \
+    fi
```

Comment:
Can we conditionally determine the ARCH and use it as a variable? so we just need one command here.

Reply:
we could pass ARCH as a build argument to drone and that should work as well.